[DOC-854] Docs for Voyager 2025.7.1 (#27816)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-voyager/release-notes.md

Voyager releases (starting with v2025.5.2) use the numbering format `YYYY.M.N`, where `YYYY` is the release year, `M` is the month, and `N` is the number of the release in that month.

## v2025.7.1 - July 1, 2025

### New feature

> - [Configuration files](../reference/configuration-file/) can now be used for the [import-data-file](../reference/bulk-data-load/import-data-file/) command in the [bulk data load](../migrate/bulk-data-load/) from files workflow.

Reviewer: ```suggestion
- You can now use [configuration files](../reference/configuration-file/) with the [import-data-file](../reference/bulk-data-load/import-data-file/) command for [bulk data load](../migrate/bulk-data-load/) from files.
```
